.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,70 @@
+# GitHub issue form. For more information see:
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+
+name: Report something is not working properly 🐛
+description:
+  Use this if there is something that is not working properly.  If you are not
+  sure or you need help making something work, please ask a question instead.
+labels: priority:❓, type:bug
+body:
+  - type: markdown
+    attributes:
+      value:
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Please tell us what happen that it shouldn't have.
+      placeholder: What happen that it shouldn't have.
```

Comment:
Grammatical correction (in the above 2 lines):
`.. what happened that shouldn't have.`